shared/menubar/remote-container.desktop.tsx

```diff
@@ -1,89 +1,93 @@
-import * as Container from '../util/container'
 import * as ConfigGen from '../actions/config-gen'
+import * as Container from '../util/container'
+import * as FsConstants from '../constants/fs'
 import * as FsGen from '../actions/fs-gen'
+import * as RPCTypes from '../constants/types/rpc-gen'
+import * as RouteTreeGen from '../actions/route-tree-gen'
+import * as SafeElectron from '../util/safe-electron.desktop'
+import * as SettingsGen from '../actions/settings-gen'
+import * as Tabs from '../constants/tabs'
+import * as Types from '../constants/types/fs'
 import Menubar from './index.desktop'
 import openUrl from '../util/open-url'
+import throttle from 'lodash/throttle'
+import {DeserializeProps} from './remote-serializer.desktop'
 import {createOpenPopup as createOpenRekeyPopup} from '../actions/unlock-folders-gen'
+import {isWindows, isDarwin, isLinux} from '../constants/platform'
 import {quit} from '../desktop/app/ctl.desktop'
-import {loginTab, AppTab} from '../constants/tabs'
-import throttle from 'lodash/throttle'
-import * as RouteTreeGen from '../actions/route-tree-gen'
-import * as SafeElectron from '../util/safe-electron.desktop'
-import * as FsConstants from '../constants/fs'
 import {urlHelper} from '../util/url-helper'
-import {isWindows, isDarwin, isLinux} from '../constants/platform'
-import * as RPCTypes from '../constants/types/rpc-gen'
-import * as SettingsGen from '../actions/settings-gen'
-import * as Types from '../constants/types/fs'
 
 const hideWindow = () => {
   SafeElectron.getRemote()
     .getCurrentWindow()
     .hide()
 }
 
-// Props are handled by remote-proxy.desktop.js
-const mapDispatchToProps = (dispatch: Container.TypedDispatch) => ({
-  _onRetrySync: (name: string) => () =>
-    // This LoadPathMetadata triggers a sync retry.
-    dispatch(FsGen.createLoadPathMetadata({path: Types.stringToPath('/keybase/private' + name)})),
-  _showUser: (username: string) => {
-    const link = urlHelper('user', {username})
-    link && openUrl(link)
-  },
-  logIn: () => {
-    dispatch(ConfigGen.createShowMain())
-    dispatch(RouteTreeGen.createNavigateAppend({path: [loginTab]}))
-  },
-  onHideDiskSpaceBanner: dispatch(FsGen.createShowHideDiskSpaceBanner({show: false})),
```

Comment:
actual bug!

Reply:
Ha, I guess this banner was just never showing >.<